src/views/directMessages/components/threadsList.js

```diff
@@ -80,7 +80,6 @@ class ThreadsList extends React.Component<Props, State> {
           hasMore={hasNextPage}
           loader={<LoadingDM />}
           useWindow={false}
-          initialLoad={false}
```

Comment:
@brianlovin this prop `initialLoad={false}` is used throughout the app
in more places—and I assume it breaks infinite scrolling after SSR in
all of those places? Why was that introduced in the first place, what's
it needed for? Removing it fixes the issue and doesn't _seem_ to break
anything, but I'm not sure if I'm just missing something...

Reply:
Honestly no clue...let's try removing it and test on alpha for a few days.